rotkehlchen/chain/base/modules/hop/balances.py

```diff
@@ -0,0 +1,19 @@
+from typing import TYPE_CHECKING
+
+from rotkehlchen.chain.evm.decoding.hop.balances import HopCommonBalances
+
+if TYPE_CHECKING:
+    from rotkehlchen.chain.base.node_inquirer import BaseInquirer
+    from rotkehlchen.db.dbhandler import DBHandler
+
+
+class HopBalances(HopCommonBalances):
+    def __init__(
+            self,
+            database: 'DBHandler',
+            evm_inquirer: 'BaseInquirer',
+    ) -> None:
+        super().__init__(
+            database=database,
+            evm_inquirer=evm_inquirer,
+        )
```

Comment:
mmmm now that I check it... the only reason to have the init is due to the typing I bleieve. Here and in the others check if this fits in one line